pkgs/development/libraries/talloc/default.nix

```diff
@@ -56,4 +57,9 @@ stdenv.mkDerivation rec {
     license = licenses.gpl3;
     platforms = platforms.all;
   };
-}
+} // stdenv.lib.optionalAttrs (stdenv.hostPlatform != stdenv.buildPlatform) {
+  # python-config from build Python gives incorrect values when cross-compiling.
```

Comment:
Is this likely to occur more often? If so, it should probably be part of a hook.

Reply:
It occurs for all samba packages that use Python: `ldb`, `talloc`, `tdb`, `tevent`, `samba` and possibly `ntdb` (it fails for other reasons that I haven't looked into). Is that enough to justify a hook?